### ISSUE TITLE
How to train ColBERT model on a lot of data? 

### ISSUE THREAD

**Issue by karynaur** [raphaelsty/neural-cherche#13]
How do i use the trainer to train on a lot of data on limited RAM on colab. Im using the V100 16GB GPU and have close to 6.9M training examples/triplets

**Comment by raphaelsty** [comment]
Hi @karynaur, here is an example google collab with a GPU (T4): https://colab.research.google.com/drive/1qQxHxZpR-NIuK1kGVBCLUmAmUm43zxJ0?usp=sharing 

I plan to add gradient accumulation soon and improve the loss but right now you should have correct results with ColBERT.
